conf/modules.config

```diff
@@ -28,5 +28,24 @@ params {
         'multiqc' {
             args = ""
         }
+       'star_genomegenerate' {
+            args             = ""
+            publish_dir      = 'genome/index'
+        }
+        'star_align' {
+            args            = "--readFilesCommand zcat --outSAMtype BAM Unsorted --outSAMunmapped Within --outBAMcompression 0 --outFilterMultimapNmax 50 --peOverlapNbasesMin 10 --alignSplicedMateMapLminOverLmate 0.5 --alignSJstitchMismatchNmax 5 -1 5 5 --chimSegmentMin 10 --chimOutType WithinBAM HardClip --chimJunctionOverhangMin 10 --chimScoreDropMax 30 --chimScoreJunctionNonGTAG 0 --chimScoreSeparation 1 --chimSegmentReadGapMax 3 --chimMultimapNmax 50"
+            publish_dir     = "alignment/${params.aligner}"
+            publish_files   = ['bam':'', 'junction':'', 'out':'log', 'tab':'log']
+        }
+        'arriba_fusion' {
+            args            = ""
+            publish_dir     = 'fusions/arriba'
+            publish_files   = ['tsv':'']
+        }
+        'starfusion' {
+            args            = "--examine_coding_effect --STAR_use_shared_memory"
```

Comment:
I am not familiar with these flags, are they new? We haven't used them before

Reply:
`--examine_coding_effect `is recommended. The second one `--STAR_use_shared_memory` is removed as I added during testing.